perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -1273,10 +1274,28 @@ sub construct_bin_request_form {
                     type         => 'Checkbox',
                     label        => $container->{name},
                     option_label => $container->{description},
+                    tags => { toggle => "form-quantity-$id-row" },
```

Comment:
Should this only be included if the max is > 1 (I don't think the JS toggle will care, though, so doesn't matter)

Reply:
Yeah, I'm happy to leave this as is.